src/seedsigner/views/seed_views.py

```diff
@@ -536,6 +535,9 @@ def run(self):
         ).display()
 
         if selected_menu_num == RET_CODE__BACK_BUTTON:
+            # If previous view is SeedOptionsView then that should be where resume_main_flow started (otherwise it would have been skipped).
+            if self.controller.back_stack[-2].View_cls == SeedOptionsView:
+                self.controller.resume_main_flow = None
```

Comment:
Clears "resume_main_flow" variable when back button goes to "SeedOptionsView" (see [here](https://github.com/SeedSigner/seedsigner/pull/286#issuecomment-1364552409) for full explanation.

Reply:
TY @EverydayBitcoiner.  I'm grabbing this snippet of code (and adding a check to avoid an IndexError) for a separate/smaller pr that fixes having to back-out-twice from AddressExplorer when coming from SeedOptionsView menu.